packages/tupaia-web/src/layout/ProjectCardList/LegacyProjectCard.tsx

```diff
@@ -14,8 +14,8 @@ const Card = styled.div`
   display: grid;
```

Comment:
I'm pretty confident that you can just delete this whole file now since it's not being used anywhere. (Also need to delete the export of it which is in ProjectCard/index.ts)